plugin-server/src/main/ingestion-queues/session-recording/utils.ts

```diff
@@ -170,6 +203,32 @@ export const parseKafkaMessage = async (
         })
     }
 
+    // this has to be ahead of the payload parsing in case we start dropping traffic from older versions
+    if (!!ingestionWarningProducer && !!teamIdWithConfig.teamId) {
+        const libVersion = readLibVersionFromHeaders(message.headers)
+        const minorVersion = minorVersionFrom(libVersion)
```

Comment:
In 12 months time we will make a version 2.0.0 - will we remember this code path (i dont think so).

Could just parse the SDK Major and minor as a float so you can just do version <= 1.74 Feels safe to ignore patch entirely but not major